tests/testthat/test-verified-ttestindependentsamples.R

```diff
@@ -8,6 +8,7 @@ context("Independent Samples TTest -- Verification project")
 options <- jaspTools::analysisOptions("TTestIndependentSamples")
 options$dependent <- "Score"
 options$group <- "Group"
+options$group.types <- "nominal"
```

Comment:
what is the current status of such changes? do we still need to update unit tests like this or is there hope that jasptools can handle it at some point?

Reply:
The current status is that it's still necessary. Hopefully, we can get rid of it once the R-syntax develops further (so we can ask JASP for the types). There are two ways to specify the types though, in the options like above, but you can also do it once for an entire dataset, see https://github.com/jasp-stats/jaspTools/blob/master/man/addTypedDataSet.Rd.